modules/nf-core/proovframe/map/meta.yml

```diff
@@ -0,0 +1,61 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "proovframe_map"
+description: frame-shift correction for long read (meta)genomics - maps proteins to reads
+keywords:
+  - frame-shift correction
+  - long-read sequencing
+  - sequence analysis
+tools:
+  - "proovframe":
+      description: "frame-shift correction for long read (meta)genomics"
+      homepage: "https://github.com/thackl/proovframe"
+      documentation: "https://github.com/thackl/proovframe"
+      tool_dev_url: "https://github.com/thackl/proovframe"
+      doi: "10.1101/2021.08.23.457338"
+      licence: ["MIT"]
+      identifier: biotools:proovframe
+
+input:
+  # Only when we have meta
```

Comment:
```suggestion
```

Reply:
is there a case where there will be no meta ?

Reply:
Naah, it was just there from the tempalte. Removed!